conf/iCubFindDependencies.cmake

```diff
@@ -65,8 +65,8 @@ checkandset_dependency(OpenCV)
 checkandset_dependency(Qt5)
 
 if(icub_firmware_shared_FOUND AND ICUB_USE_icub_firmware_shared)
-  if(icub_firmware_shared_VERSION VERSION_LESS 1.34.3)
-    message(FATAL_ERROR "An old version of icub-firmware-shared has been detected: at least 1.34.3 is required")
+  if(icub_firmware_shared_VERSION VERSION_LESS 1.34.4)
+    message(FATAL_ERROR "An old version of icub-firmware-shared has been detected: at least 1.34.4 is required")
```

Comment:
The latest released tag is 1.35.0.
Therefore, I suppose this needs to be updated somwhow.